### DISCUSSION TITLE
Error with command Line

### DISCUSSION THREAD

**Discussion by temium62** [simulot/immich-go#291] [Q&A] [answered]
Hello
thank you for your appli
I use Windows in French : when I copy the command i have an Error 
immich-go : Le terme «immich-go» n'est pas reconnu comme nom d'applet de commande, fonction, fichier de script ou
programme exécutable. Vérifiez l'orthographe du nom, ou si un chemin d'accès existe, vérifiez que le chemin d'accès
est correct et réessayez.
Au caractère Ligne:1 : 1
    + CategoryInfo          : ObjectNotFound: (immich-go:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Any idea to resolve this

**Comment by simulot** [comment, marked as answer]
Bonjour, 
Powershell need a path to a command.
Try again with **./** before the program name.

```
./immich-go -server=xxxx -key=kkkkk upload...
```

**Reply by temium62** [reply]
Thank you, it's ok .
Merci beaucoup ça fonctionne ;-)
